app/templates/components/events/view/overview/event-setup-checklist.hbs

```diff
@@ -0,0 +1,56 @@
+<div class="content">
+  <div class="header">{{t 'Event setup checklist'}}</div>
+</div>
+<div class="ui segments content">
+  <div class="ui teal inverted segment">
+    <i class="large checkmark middle aligned icon left floated"></i>
```

Comment:
should I use big icon or huge icon, I tried using huge icon for check mark but it appears very big related to the content.